src/building/suggested.md

```diff
@@ -87,6 +87,32 @@ You can also use `--keep-stage 1` when running tests. Something like this:
 - Initial test run: `./x.py test -i --stage 1 src/test/ui`
 - Subsequent test run: `./x.py test -i --stage 1 src/test/ui --keep-stage 1`
 
+## Fine-tuning debug and optimizations
+
+By default, `config.toml` sets `debug = true`. Unless you plan to run a
```

Comment:
I think the default is `debug = false`, right?

Also, it might make sense to update the following references with the new advice, if you don't mind:
```
src/compiler-debugging.md
48:If you want line numbers for the stack trace, you can enable `debug = true` in
50:line numbers, but `debug = true` gives full debuginfo). Then the backtrace will

src/getting-started.md
129:- `debug = true`: enables debug symbols and `debug!` logging, takes a bit longer to compile.

src/building/how-to-build-and-run.md
46:# `debug-assertions = true`. Note that `debug = true` will make your build
49:debug = true
```

Reply:
Hmm, that would be updating a significant chunk of advice, but I have no clue if what I suggest is correct. I would need approval by someone who knows that stuff before merging that

Reply:
At this point I'm inclined to remove the first paragraph of advice and leave only the thing about crate-wise optimizations

Reply:
@Nadrieril Sure, that makes sense. There will be more updates to the recommendations and x.py defaults soon, I think...

Reply:
I did that